core/unix/include/syscall_linux_x86.h

```diff
@@ -356,10 +358,29 @@
 #    define __NR_statx 332
 #    define __NR_io_pgetevents 333
 #    define __NR_rseq 334
+#    define __NR_pidfd_send_signal 424
```

Comment:
This gap looks suspicious: but on inspection it seems to be deliberate.

Reply:
`__NR_pidfd_send_signal` onwards syscall number assignments seem to align on 32- and 64-bit, and the space is perhaps for arch-specific syscall. https://github.com/torvalds/linux/commit/0d6040d4681735dfc47565de288525de405a5c99